tests/devices/unit_tests/test_apple2_undulator.py

```diff
@@ -18,13 +19,28 @@
 from dodal.devices.apple2_undulator import (
     DEFAULT_MOTOR_MIN_TIMEOUT,
     Apple2PhasesVal,
+    MotorWithoutStop,
     UndulatorGap,
     UndulatorGateStatus,
     UndulatorJawPhase,
     UndulatorPhaseAxes,
 )
 
 
+@pytest.fixture(scope="function")
+def logger(caplog: pytest.LogCaptureFixture):
+    logger = getLogger()
+    _ = [logger.removeHandler(h) for h in logger.handlers if h != caplog.handler]  # type: ignore
+    return logger
```

Comment:
I don't think this is necessary, we only instantiate the local logging Handler in tests/dodal connect. Don't need to worry about forwarding to graylog unexpectedly.
```suggestion
```